lib/tedious/connection-pool.js

```diff
@@ -13,7 +13,8 @@ class ConnectionPool extends BaseConnectionPool {
       server: this.config.server,
       options: Object.assign({
         encrypt: typeof this.config.encrypt === 'boolean' ? this.config.encrypt : true,
-        trustServerCertificate: typeof this.config.trustServerCertificate === 'boolean' ? this.config.trustServerCertificate : false
+        trustServerCertificate: typeof this.config.trustServerCertificate === 'boolean' ? this.config.trustServerCertificate : false,
+        tdsVersion: this.config.tdsVersion
```

Comment:
I'm not sure this should be a first class property on our config if it is only a property that is accepted by the tedious driver.

It doesn't look like this change is actually required given the other implementation.